authn.js

```diff
@@ -257,6 +311,24 @@ function setup(app) {
   });
 }
 
+
+/**
+ * Creates a user record from the given `idToken` after verifying it.
+ *
+ * @param {String} idToken
+ * @param {String} client. Optional. Passed to `verifyToken()`.
```

Comment:
I was initially confused by the type hinting here since the array `BEARER_COGNITO_CLIENT_IDS` is being passed in as `client`. I think it would be clearer for the type hint to be `{String|String[]}`.

Reply:
Good catch! Updated.